packages/@webex/plugin-cc/src/services/config/types.ts

```diff
@@ -22,6 +21,11 @@ export const CC_EVENTS = {
   AGENT_BUDDY_AGENTS_RETRIEVE_FAILED: 'BuddyAgentsRetrieveFailed',
 } as const;
 
+export type WelcomeEvent = {
```

Comment:
Is there a reason we have moved this event here - just curious.

Reply:
Got some circular dependencies, so i had to move it.